cpp/zimt/dtw.cc

```diff
@@ -77,12 +77,23 @@ float HwyDeltaNorm(hwy::Span<const float> span_a, hwy::Span<const float> span_b,
   const Vec order_vec = Set(d, order);
   const Vec max_reciprocal = Div(Set(d, 1), Set(d, max));
   double sum = 0;
-  for (size_t index = 0; index < span_a.size(); index += Lanes(d)) {
-    const Vec delta =
-        Sub(Load(d, span_a.data() + index), Load(d, span_b.data() + index));
-    const Vec downscaled_values = Mul(delta, max_reciprocal);
-    const Vec pows = Exp(d, Mul(order_vec, Log(d, downscaled_values)));
-    sum += static_cast<double>(ReduceSum(d, pows));
+  if (order == 2.0) {
```

Comment:
You can actually remove the order parameter, we'll only use 2-norm for dtw.

Reply:
let's do it in another PR